pytorch_lightning/callbacks/model_checkpoint.py

```diff
@@ -545,6 +578,7 @@ def _save_last_checkpoint(self, trainer, pl_module, ckpt_name_metrics):
 
         # when user ALSO asked for the 'last.ckpt' change the name
         if self.save_last:
+            rank_zero_info("Saving latest checkpoint...")
```

Comment:
This message will appear every time the checkpoint logic is run.

We only want it to appear the last time (as it did before)

Reply:
I've added a condition to call this info only in `on_training_end`